bin/newrelic-admin

```diff
@@ -52,11 +52,11 @@ Usage: newrelic-admin run-perl [--tx <transaction-name>] ...
 Executes the Perl interpreter with the supplied arguments but forces
 the initialisation of the agent and wraps the command in a transaction.
 
-If using an agent configuration file the path to the file should be
+If using an agent configuration file, the path to the file should be
 supplied by the environment variable NEWRELIC_CONFIG_FILE. Alternatively,
 just the licence key, application and log file details can be supplied via
-environment variables NEWRELIC_LICENSE_KEY, NEWRELIC_APP_NAME and
-NEWRELIC_LOG_FILE.
+environment variables NEWRELIC_LICENSE_KEY, NEWRELIC_APP_NAME, NEWRELIC_LOG_FILE
+and NEWRELIC_LOG_LEVEL.
```

Comment:
If you're adding this here, the beginning of this sentence needs to change as well.
It might be that a better choice would be to point people in the direction of [the POD for the config object](https://github.com/cv-library/NewFangle-Agent/blob/main/lib/NewFangle/Agent/Config.pod) and tell them that any variable that _it_ can read have an effect on this side as well

Reply:
I'll switch to using the config file, it's a good idea

Reply:
The phrasing here is still off. The sentence starts saying that "just A, B and C can be set" but then we list 4 variables. We can solve this by being more general and just telling people that settings that affect the config object will affect this too.